vars/tasks.groovy

```diff
@@ -140,6 +140,7 @@ def functionalXpack(Map params = [:]) {
       'x-pack/test/security_solution_cypress/',
       'x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/',
       'x-pack/plugins/triggers_actions_ui/public/application/context/actions_connectors_context.tsx',
+      'x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/',
```

Comment:
Hey I don't think this file is in use anymore since the CI changed from jenkins to buildkite. Can you add it here instead please? https://github.com/elastic/kibana/blob/main/.buildkite/scripts/pipelines/pull_request/pipeline.ts#L61-L82